dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/utils/DataxUtilsTest.java

```diff
@@ -84,9 +99,10 @@ public void testConvertKeywordsColumns() throws Exception {
      */
     @Test
     public void testDoConvertKeywordsColumn() throws Exception {
-        String fromColumn = " \"`select`\" ";
-        String targetColumn = "`select`";
-
-        assertEquals(DataxUtils.doConvertKeywordsColumn(DbType.MYSQL, fromColumn), targetColumn);
+        assertEquals(DataxUtils.doConvertKeywordsColumn(DbType.MYSQL, " \"`select`\" "), "`select`");
+        assertEquals(DataxUtils.doConvertKeywordsColumn(DbType.POSTGRESQL, " \"`select`\" "), "\"select\"");
```

Comment:
assertEquals first param is expected value, sceond param is actual value, please swap there params, from 102-105 lines.